lib/suse/toolkit/utilities.rb

```diff
@@ -21,7 +21,7 @@ def system_auth
 
       rescue
         raise SUSE::Connect::CannotBuildBasicAuth,
-              "Cannot read username and password from #{SUSE::Connect::Credentials::GLOBAL_CREDENTIALS_FILE}"
+              "\nCannot read username and password from #{SUSE::Connect::Credentials::GLOBAL_CREDENTIALS_FILE}."
```

Comment:
I think you should better use `SUSE::Connect::Credentials.system_credentials_file` here